aws/chef-bootstrap.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+# Code-dot-org Chef bootstrap script.
+# One-liner to run:
+# aws s3 cp s3://cdo-dist/chef/bootstrap.sh - | sudo bash -s -- [options]
+#
+# Options:
```

Comment:
This script is > 100 lines and fairly complex, would it make sense for it to be written in Ruby?  Most of our developers are more familiar with Ruby and it's more powerful than Bash.